stardis/io/model/tests/test_model_io.py

```diff
@@ -60,47 +46,47 @@ def test_mesa_truncation(mesa_model):
     assert len(mesa_model.data) == len_before_truncation - 1
 
 
-def test_read_marcs_model_scaled_log_number_fraction(marcs_model_test_data_file_path):
+def test_read_marcs_model_scaled_log_number_fraction(marcs_model_file_path):
```

Comment:
Have these functions take in marcs_model so that you don't have to redefine model by calling the read_marcs_model() function multiple times. 

Also I think you deleted marcs_model_file_path to use the constant instead, so the tests will probably break on this.

Reply:
I just pushed the changes that I think should fix this